apps/prairielearn/src/lib/manualGrading.ts

```diff
@@ -113,15 +114,15 @@ export async function nextUngradedInstanceQuestionUrl(
 
 /** Populates the locals objects for rubric data. Assigns values to `rubric_data` in the locals.
  *
- * @param {Record<string, any>} locals - The locals data to be retrieved and updated. The `assessment_question` is expected to have been retrieved before this call, as well as any value that impacts the mustache rendering, such as `variant` and `submission`.
- * @returns {Promise<void>}
+ * @param locals - The locals data to be retrieved and updated. The `assessment_question` is expected to have been retrieved before this call, as well as any value that impacts the mustache rendering, such as `variant` and `submission`.
  */
-export async function populateRubricData(locals) {
+export async function populateRubricData(locals: Record<string, any>): Promise<void> {
   // If there is no assessment question (e.g., in question preview), there is no rubric
   if (!locals.assessment_question) return;
 
+  let rubric_data: RubricData | null = null;
```

Comment:
Nit: I believe this function could be cleaned up with an early return if `locals.assessment_question.manual_rubric_id` isn't present.

Reply:
Done in 1f1ab79c6.